### PR TITLE
Improve the kick off point for actions from ListMarkets

### PR DESCRIPTION
**Pull request by ptare**
Issue #1040 

Signed-off-by: Prasad Tare <prasad.tare@gmail.com>

### CODE REVIEW

File: core/scripts/cli/sponsor/showMarketDetails.js [hunk 1]
```diff
@@ -28,15 +29,30 @@ const showMarketDetails = async (web3, artifacts, emp) => {
...
-      await create(web3, artifacts, emp);
+    const input = (await inquirer.prompt(prompt))["choice"];
+    switch (input) {
+      case actions.create:
+        await create(web3, artifacts, emp);
+        break;
+      case actions.redeem:
+        await redeem(web3, artifacts, emp);
+        break;
+      case actions.back:
```
Comment (nicholaspai) [inline core/scripts/cli/sponsor/showMarketDetails.js]: I like the `switch...case...case` structure much better for this, but I see why you didn't have it before when there were only 2 choices

File: core/scripts/cli/sponsor/showMarketDetails.js [hunk 2]
```diff
@@ -28,15 +29,30 @@ const showMarketDetails = async (web3, artifacts, emp) => {
       "Collateral:",
       fromWei(collateral.toString())
     );
+
+    const actions = {
+      back: "Back",
```
Comment (nicholaspai) [inline core/scripts/cli/sponsor/showMarketDetails.js]: do you still need the `backAction` variable declaration on line 11?

Comment (ptare) [reply core/scripts/cli/sponsor/showMarketDetails.js]: Do you mind if I delete it in a follow up? Getting this PR in would unblock both of us I guess.

Comment (nicholaspai) [reply core/scripts/cli/sponsor/showMarketDetails.js]: sure

**Pull request merged**